src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx

```diff
@@ -87,85 +87,79 @@ The service map page provides a visual representation of your entire architectur
-- `http.status_code` from [http span][] semantic conventions
-- `undefined` if none of the above are present
+* Span status description (for example, `otel.status_description`)
+* `rpc.grpc.status_code` from [rpc span](https://opentelemetry.io/docs/specs/semconv/rpc/rpc-spans/) semantic conventions
+* `http.status_code` from [http span](https://opentelemetry.io/docs/specs/semconv/http/http-spans/) semantic conventions
+* `undefined` if none of the above are present
 
 ## Page: Metrics explorer [#metrics-explorer-page]
 
-The Metrics explorer provides tools for exploring a service's metrics in a generic manner. See [explore your data](https://docs.newrelic.com/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/#explore-data) for more information.
+The Metrics explorer provides tools for exploring a service's metrics in a generic manner. See [explore your data](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/#explore-data) for more information.
 
 ## Golden signals [#golden-signals]
 
 The golden signals of throughput, response time, and error rate appear in several places throughout the OpenTelemetry APM UI. When used, they are computed as follows:
 
-For **metrics**, the queries assume data conforms to the [http metric][] or [rpc metric][] semantic conventions.
+For metrics, the queries assume data conforms to the [http metric](https://opentelemetry.io/docs/specs/semconv/http/http-metrics/)or [rpc metric](https://opentelemetry.io/docs/specs/semconv/rpc/rpc-metrics/) semantic conventions.
 
-For **spans**, the queries are generic, utilizing only the top level span data model. Spans are counted towards throughput and response time if they are root entry spans into a service, computed using a heuristic of `WHERE span.kind = server OR span.kind = consumer`. Spans are errors if they have a status code of `ERROR` (i.e. `otel.status_code = ERROR`).
+For spans, the queries are generic, utilizing only the top level span data model. Spans are counted towards throughput and response time if they are root entry spans into a service, computed using a heuristic of `WHERE span.kind = server OR span.kind = consumer`. Spans are errors if they have a status code of `ERROR` (for example, `otel.status_code = ERROR`).
 
 ## Narrow data with filters [#narrow-with-filters]
 
-Several pages include a filter bar (i.e. **Narrow data to..**). This allows you to filter queries on the page to match the criteria. For example, you might narrow to a particular canary deployment by filtering for `service.version='1.2.3-canary'`. Filters are preserved when navigating between pages.
+Several pages include a filter bar, with options like **Narrow data to...**. This allows you to filter queries on the page to match the criteria. For example, you might narrow to a particular canary deployment by filtering for `service.version='1.2.3-canary'`. Filters are preserved when navigating between pages.
```

Comment:
@jack-berg Would "**Sort by**" be better than "**Narrow data to..**"? When clicking around in a Demogorgon app, I don't see the "Narrow data to" option at the top of any APM UI pages.

Reply:
I'm not familiar with the Demogorgon app, but the UI component I'm referencing does indeed use the language "Narrow data to". See screenshot:
<img width="1636" alt="Screenshot 2024-05-24 at 10 02 33 AM" src="https://github.com/newrelic/docs-website/assets/34418638/ee8358e6-086a-42c4-b5e8-f35420d4680d">
Let's sync up to take a look at Demogorgon to make sure we're looking at the same type of app.

Reply:
Oops, I was not looking at an OTEL service. Thanks for the screenshot!